Vindicta.Altis/AI/Commander/AICommander.sqf

```diff
@@ -3049,4 +3049,8 @@ http://patorjk.com/software/taag/#p=display&f=Univers&t=CMDR%20AI
 		true
 	} ENDMETHOD;
 
-ENDCLASS;
\ No newline at end of file
+ENDCLASS;
+
+AI_fnc_addActivity = {
```

Comment:
Sorry, why do we need to add this? All the OOP functions are compiled into format `className_fnc_methodName`, same for static methods.

Reply:
The problem is I can't use macros inside the stringified code that triggers use.
I could access it directly like you say, that but it introduces a dependency on that naming convention never changing. This just abstracts that problem for minimal cost (function is called infrequently).
I could perhaps use format on the stringified code to inject the macro generated name, its a bit noisey in the code but would work.

Reply:
Yeah ok, I agree then, naming convention might change in future, but IDK why it would change, but still....